targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_MODULE_UBLOX_ODIN_W2/TARGET_MTB_UBLOX_ODIN_W2/PinNames.h

```diff
@@ -148,16 +148,14 @@ typedef enum {
     //P_TP11,         // BOOT0
 
     // Internal
-    LED1       = PD_9,
-    LED2       = PA_12,
-    LED3       = PD_8,
-    LED4       = PA_11,
-    LED5       = PC_2,
-    LED6       = PA_3,
-    LED7       = PF_6,
     LED_RED    = PE_0,
     LED_GREEN  = PB_6,
     LED_BLUE   = PB_8,
+
+    LED1       = LED_RED,
+    LED2       = LED_GREEN,
+    LED3       = LED_BLUE,
```

Comment:
LED4 not needed? I recall some tests required LED1-4. I can not locate those tests (my searching fails), but most of the platforms defines LED1-4.

Reply:
@0xc0170 , I think LED1, LED2 are mandatory for the tests.. not sure about LED3 & LED4.. 
@cmonr .. IIRC, you added an LED2 in one of my other PR's for the tests as mandatory.. are LED3 and LED4 also mandatory?

Reply:
> @cmonr .. IIRC, you added an LED2 in one of my other PR's for the tests as mandatory.. are LED3 and LED4 also mandatory?

I assume not, wanted to be certain as it was previously LED up to 7, now we reducing to 3.

Reply:
At the moment, only LED1 and LED2 are manditory for the tests to run (which now that I'm thinking about it, that should probaly be a bug). All other LED definitions are not necessary.

Reply:
Hmm, this maybe something to consider for future. @cmonr can you point us to the location(s) where the LED's are used for the tests..My search yielded nothing..